src/Graphics/Vty/Input/Events.hs

```diff
@@ -43,6 +43,7 @@ data Event
     -- row. Some terminals report only that a button was released
     -- without specifying which one; in that case, Nothing is provided.
     -- Otherwise Just the button released is included in the event.
+    -- Mouse wheel release events are not reported.
```

Comment:
On mice with wheels it is also possible to press them in many cases, but since (as far as I can tell) this patch only detects wheel rotation, not pressing, the comment above should also mention that wheel presses are also not reported (unless they are sent as a `BMiddle`, which will be up to the terminal emulator).